app/src/main/res/values/strings.xml

```diff
@@ -235,6 +235,7 @@
     <string name="feedback_subject">Feedback</string>
 
     <string name="feedback_safe">Safe address: %1$s</string>
+    <string name="feedback_chain">Chain: %1$s (%2$s)</string>
```

Comment:
Will it be "Chain:  (nil)" when no safe is selected @elgatovital ?

Reply:
No it is `Chain: (null)` :-)